.github/workflows/test-build.yml

```diff
@@ -0,0 +1,17 @@
+name: Test build
+on: [pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    container: trion/ng-cli-karma
```

Comment:
container trion/ng-cli-karma necessary? Will this run without a specialized container?